playbook/workshops/dodge/blank_canvas.md

```diff
@@ -0,0 +1,204 @@
+understand what it does:
+
+> ![](img/t1_background_color_change.gif)
+
+## Understanding `createCanvas`
+
+```
+This code sets the background color of the canvas:
+
+  background(255, 255, 255);
+              ^    ^    ^
+              |    |    └ set how much *red* there is
+              |    |    
+              |    └ set how much *green* there is
+              |
+              └ set how much *blue* there is
+```
+
+## Setting the Proper Dimensions
+
+Now that you've gotten a chance to experiment, we want our canvas to have a
+width of 300 and a height of 500.
+
+![](img/checkmark.png) Go ahead and make sure that your setup function looks
+like this:
+
+```js
+function setup() {
+  createCanvas(300, 500);
+}
+```
+
+## Recap
+
+- You learned how to use `createCanvas`
+- You learned how to use `background`
+
+## Next Up
+
+| **[![](img/sq_2_add_player_sprite.png)    <br> 2: Adding the Player Sprite]    (add_player_sprite.md)**    |
+|:----------------------------------------------------------------------------------------------------|
+
+---
+
+# Appendix
+
+## Documentation ![](img/documentation.png)
+
+**Links to the official p5.js Documentation**
+
+- [`background(255, 255, 255)`](http://p5js.org/reference/#/p5/background)
+- [`createCanvas(300, 500);`](http://p5js.org/reference/#/p5/createCanvas)
+
+## Computer Science Fundamentals ![](img/computer_science.png)
+
+**_This Section is Coming Soon!_**—In the mean time, here is a sneak peak:
+
+### Functions
+
+`createCanvas` and `background` are what we call _**functions**_.
+
+Functions can have the following:
+
+- inputs
+- side effects
+- outputs
+
+#### Lemonade Analogy
+
+Consider the lemonade making machine as a function:
+
+![](img/function_machine.png)
+
+In the above drawing, you can see that
+
+- the inputs are a glass, lemon, and mint
+- the output is lemonade
+
+But what are the side effects?
+
+- well the lemonade juicing machine gets hot from use
+- you, the user, who is not an input get better at learning how to make lemonade
+- you and your friends bond over making lemonade
+
+Sometimes, you care more about the side effects than the outputs.
+
+#### Create Canvas
+
+Let's examine the components of `background(255, 255, 255)`
+
+##### Inputs
+
+The inputs are whatever in the parenthesis.
+
+In this case the `background` function has 3 inputs:
+
+```
+background(255, 255, 255);
+            ^    ^    ^
+            |    |    └ input 3
+            |    |    
+            |    └ input 2
+            |
+            └ inputs 1
+```
+
+##### Side Effects
+
+The side effect of this function is to set the color of the canvas.
+
+##### Outputs
+
+This function has no outputs. This may not seem like it makes sense right now
+but if you visit the functions section of createSprite (which doesn't exist
+yet), it will make much more sense.
+
+## Associated Modules:
+
+**_This Section is Coming Soon!_**—We don't have any modules yet, but we're working on it!
```

Comment:
Same thing here. It'd be awesome if you could use regular dashes instead of em or endashes!